chsdi/templates/htmlpopup/base.mako

```diff
@@ -53,7 +53,7 @@
 <body>
   <div class="chsdi-htmlpopup-container">
 % else:
-  <div class="htmlpopup-container">
+  <div id="${c['layerBodId']}#${c['featureId']}" class="htmlpopup-container">
```

Comment:
yes. Otherwise, we would not be capable to get the geometry for a feature. We construct the url using layerBodId and featureId. Never faced a problem because of that, AFAIK.